Source/carthage/Build.swift

```diff
@@ -31,6 +31,12 @@ public struct BuildCommand: CommandType {
 
 				let (stdoutSignal, schemeSignals) = self.buildProjectInDirectoryURL(directoryURL, options: options)
 				stdoutSignal.observe { data in
+					if options.verbose {
+						if let outString = NSString(data: data, encoding: NSUTF8StringEncoding) {
+							carthage.print(outString)
```

Comment:
Instead of logging to two different places, maybe `--verbose` should just dump everything to the terminal. In that case, could we perhaps skip the temporary file, and use `stdoutHandle = NSFileHandle.fileHandleWithStandardOutput()`?